src/NewTwitchApi/Resources/TeamsApi.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NewTwitchApi\Resources;
+
+use GuzzleHttp\Exception\GuzzleException;
+use Psr\Http\Message\ResponseInterface;
+
+class TeamsApi extends AbstractResource
+{
+    /**
+     * @throws GuzzleException
+     * @link https://dev.twitch.tv/docs/api/reference#get-channel-teams
+     */
+    public function getChannelTeams(string $bearer, string $broadcasterId): ResponseInterface
+    {
+        $queryParamsMap = [];
+
+        $queryParamsMap[] = ['key' => 'broadcaster_id', 'value' => $broadcasterId];
+
+        return $this->getApi('teams/channel', $bearer, $queryParamsMap);
+    }
+
+    /**
+     * @throws GuzzleException
+     * @link https://dev.twitch.tv/docs/api/reference#get-teams
+     */
+    public function getTeams(string $bearer, string $name = null, string $id = null): ResponseInterface
+    {
+        $queryParamsMap = [];
+
+        if ($name) {
+            $queryParamsMap[] = ['key' => 'name', 'value' => $name];
+        }
+
+        if ($id) {
+            $queryParamsMap[] = ['key' => 'id', 'value' => $id];
+        }
+
+        return $this->getApi('teams', $bearer, $queryParamsMap);
+    }
+    
```

Comment:
Theres some kind of weird tab or space here causing the Coding Standards check to fail. Just run `php-cs-fixer fix` on this file and it should clean it up for you and let the test pass.

Reply:
Oh I see